core-dev/packages/coq.8.7.dev/opam

```diff
@@ -12,8 +12,7 @@ build: [
     "-prefix" prefix
     "-usecamlp5"
     "-camlp5dir" "%{lib}%/camlp5"
-    "-coqide" "no"
-    "-debug"]
+    "-coqide" "no"]
```

Comment:
The official Coq package is compiled with `-debug`. Assuming that is correct, there is no reason for the unofficial ones to be compiled without it.

Reply:
It's now the default